### PR TITLE
Added new test and other minor adjustments.

### PR DESCRIPTION
**Pull request by PauloGoncalvesBH**
Added test to 'setTimeout' method. If already had the test, the absence of 'constants' adjusted would be seen in the tests.

 Now the tests are executed with random order to guarantee the independence of tests.

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -36,9 +36,10 @@ Started
 ..................
 
 
-21 specs, 0 failures
-Finished in 2.968 seconds
+22 specs, 0 failures
+Finished in 1.96 seconds
```
Comment (PauloGoncalvesBH) [inline docs/CONTRIBUTING.md]: This time is according to the result of the Semaphore build

File: test/constants.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ const constants = {
   SAMPLE_URL: "http://example.com/",
   SHORTEN_URL: "https://goo.gl/Us9Txw",
   TEXT_NOT_EXISTING_ON_VISIBLE_ELEMENTS: "foo",
+  TIMEOUT : 500,
```
Comment (wlsf82) [inline test/constants.js]: What about calling it `TIMEOUT_IN_MS`?

Comment (PauloGoncalvesBH) [reply test/constants.js]: I liked the suggestion, I will change it.

Comment (PauloGoncalvesBH) [reply test/constants.js]: Done.

**Pull request merged**